attributes/default.rb

```diff
@@ -510,6 +510,7 @@
 default['cluster']['raid_vol_ids'] = nil
 default['cluster']['dns_domain'] = nil
 default['cluster']['use_private_hostname'] = 'false'
+default['cluster']['add_node_hostnames_in_hosts_file'] ||= node['cluster']['use_private_hostname']
```

Comment:
if `use_private_hostname` is set to true but we force `add_node_hostnames_in_hosts_file` to false don't we want the end result to be `false`?

Reply:
I fixed this by using a singal `=`. I manually tested it.